modules/custom/wxt_ext/wxt_ext_fontawesome/wxt_ext_fontawesome.info.yml

```
@@ -0,0 +1,9 @@
+name: 'WxT Extend Fontawesome'
+description: 'The most popular icon set and toolkit on the web.'
```

```suggestion
description: 'Add Font Awesome to the site.'
```

I don't think we should be making that type of statement.